QueueIntegrations/Prognetics.Saga.Queue.RabbitMQ/Consuming/IRabbitMQConsumersFactory.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Prognetics.Saga.Orchestrator.Contract;
+﻿using Prognetics.Saga.Core.Model;
+using Prognetics.Saga.Orchestrator.Contract;
 using RabbitMQ.Client;
 
 namespace Prognetics.Saga.Queue.RabbitMQ.Consuming;
```

Comment:
Please move to abstraction layer. Or specific folder if it's not the case here as it's MQ specific.